### PR TITLE
paru: replace -P example by 2 --getpkgbuild examples

### PR DESCRIPTION
**Pull request by agstrc**
Change "paru -P --stats", which no longer works, to a new, and also useful, example.

- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by agstrc** [comment]
Example showing that the previous command no longer works:
example

**Review by bl-ue — APPROVED** [review]
Nice.  @agstrc while you're here, do you mind updating the more information link to begin with a capital 'M' like this:
```diff
- https://github.com/morganamilo/paru
+ https://github.com/Morganamilo/paru
```
That's the real URL.

**Review by owenvoke — APPROVED** [review]

**Comment by bl-ue** [comment]
Pinging @navarroaxel as he's an (the?) arch master @tldr-pages 🙂

File: pages/linux/paru.md [hunk 1]
```diff
@@ -19,6 +19,6 @@
 
 `paru -Si {{package_name}}`
 
-- Show statistics for installed packages and system health:
+- Download PKGBUILDs from the AUR or ABS:
```
Comment (navarroaxel) [inline pages/linux/paru.md:22]: This download all the package source files not only the `PKGBUILD`.
```suggestion
- Download `PKGBUILD` and other package source files from the AUR or ABS:
```
@bl-ue ?

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Comment by agstrc** [comment]
Just pushed the requested changes

**Review by navarroaxel — APPROVED** [review]

File: pages/linux/paru.md [hunk 2]
```diff
@@ -19,6 +19,10 @@
 
 `paru -Si {{package_name}}`
 
-- Show statistics for installed packages and system health:
+- Download `PKGBUILD` and other package source files from the AUR or ABS:
 
-`paru -P --stats`
+`paru --getpkgbuild  {{package_name}}`
+
+- Display the `PKGBUILD` of a package:
```
Comment (bl-ue) [inline pages/linux/paru.md:26]:
```suggestion
- Display the `PKGBUILD` file of a package:
```

**Review by bl-ue — APPROVED** [review]
One minor change, but looks good otherwise 👍🏻

**Comment by bl-ue** [comment]
Welcome to tldr-pages, @agstrc! 🙂

**Pull request merged**